addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -31,8 +14,8 @@
                             <field name="pricing_type" invisible="1" widget="radio"/>
                             <field name="timesheet_product_id" string="Default Service" invisible="1" context="{'default_detailed_type': 'service', 'default_service_policy': 'delivered_timesheet', 'default_service_type': 'timesheet'}"/>
                             <field name="sale_order_id" invisible="1" options="{'no_create': True, 'no_edit': True, 'delete': False, 'no_open': True}"/>
-                            <field name="sale_line_id" groups="!sales_team.group_sale_salesman" string="Default Sales Order Item" options="{'no_create': True, 'no_edit': True, 'delete': False, 'no_open': True}"/>
-                            <field name="sale_line_id" groups="sales_team.group_sale_salesman" string="Default Sales Order Item" options="{'no_create': True, 'no_edit': True, 'delete': False}"/>
+                            <field name="sale_line_id" groups="!sales_team.group_sale_salesman" string="Sales Order Item" options="{'no_create': True, 'no_edit': True, 'delete': False, 'no_open': True}"/>
+                            <field name="sale_line_id" groups="sales_team.group_sale_salesman" string="Sales Order Item" options="{'no_create': True, 'no_edit': True, 'delete': False}"/>
```

Comment:
The string here can be removed since it is equal to the string defined in the field definition in python. 🙂 
```suggestion
                            <field name="sale_line_id" groups="!sales_team.group_sale_salesman" options="{'no_create': True, 'no_edit': True, 'delete': False, 'no_open': True}"/>
                            <field name="sale_line_id" groups="sales_team.group_sale_salesman" options="{'no_create': True, 'no_edit': True, 'delete': False}"/>
```